src/components/HomePage.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="flex flex-row justify-center">
+    <div class="hidden lg:flex lg:w-1/4" @click="clickAway"></div>
+    <div class="max-w-[610px] flex flex-col lg:w-2/4 w-full">
+      <PageHeader @show-form="showToDoForm" />
+      <div v-if="showForm">
+        <ToDoItemComp
```

Comment:
Please refer to styleguide, give clear and understandable names to components, like TodoItemCard, or EditableTodo or something similar